### PR TITLE
mosh: libutempter is now in illumos-gate

### PR DESCRIPTION
**Pull request by AndWac**

### CODE REVIEW

File: components/shell/mosh/Makefile [hunk 1]
```diff
@@ -36,29 +37,26 @@ COMPONENT_LICENSE=      GPLv3+
...
 COMPONENT_SUMMARY=	Mosh (mobile shell) - Remote terminal application that allows roaming, supports intermittent connectivity, and provides intelligent local echo and line editing of user keystrokes (replacement for SSH).
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+TEST_TARGET=		$(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
 
-CONFIGURE_BINDIR.64= $(CONFIGURE_BINDIR.32)
+LDFLAGS += -lutempter
```
Comment (pyhalov) [inline components/shell/mosh/Makefile]: should we just configure mosh with `--with-utempter`?

Comment (AndWac) [reply components/shell/mosh/Makefile]: That would also work, the only difference I see here is with gmake REQUIRED_PACKAGES:
> REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
71d72
< REQUIRED_PACKAGES += system/library/gcc-4-runtime

("LDFLAGS += -lutempter" version determines the first dependency, "CONFIGURE_OPTIONS+= --with-utempter" the second one.)

**Comment by AndWac** [comment]
How to proceed? Shall I cange to configure with --with-utempter?

**Comment by pyhalov** [comment]
Configure --with-utemper

**Comment by pyhalov** [comment]
Also look at https://github.com/OpenIndiana/oi-userland/pull/5963

**Pull request merged**